### PR TITLE
[DB-231] [risk=no] updating chart titles and synonyms header

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Db-231, 232, 233, 236 related changes.

### CODE REVIEW

File: public-ui/src/app/data-browser/chart/chart.component.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export class ChartComponent implements OnChanges {
   @Input() chartType: string;
   @Input() sources = false;
   @Input() genderId: string; // Hack until measurement design of graphs gender overlay
+  @Input() component: string;
```
Comment (calbach) [inline public-ui/src/app/data-browser/chart/chart.component.ts]: nit: `component` has too strong of a meaning in Angular, would be better to use something like `category`, or `domainType`?

File: public-ui/src/app/views/survey-view/survey-view.component.ts [hunk 2]
```diff
@@ -35,6 +35,7 @@ export class SurveyViewComponent implements OnInit, OnDestroy {
   conceptCodeTooltip: any;
   binnedSurveyQuestions: string[] = ['1585864', '1585870', '1585873', '1585795', '1585802',
     '1585820', '1585889', '1585890'];
+  component = 'surveys';
```
Comment (calbach) [inline public-ui/src/app/views/survey-view/survey-view.component.ts]: nit: I don't think there's much value to setting this variable, I'd probably just inline it

File: public-ui/src/app/data-browser/chart/chart.component.ts [hunk 3]
```diff
@@ -528,5 +530,14 @@ export class ChartComponent implements OnChanges {
     };
 
   }
+  public getAgeChartTitle(component: string) {
+    if (component === 'ehr') {
```
Comment (calbach) [inline public-ui/src/app/data-browser/chart/chart.component.ts]: nit: seems like a good use case for an enum

Comment (SrushtiGangireddy) [reply public-ui/src/app/data-browser/chart/chart.component.ts]: Changed.

File: public-ui/src/app/views/survey-view/survey-view.component.ts (hunk 2 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/views/survey-view/survey-view.component.ts]: Changed it to inline.

File: public-ui/src/app/data-browser/chart/chart.component.ts (hunk 1 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/data-browser/chart/chart.component.ts]: Changed it to domain type thanks.

File: public-ui/src/app/data-browser/concept-charts/concept-charts.component.html [hunk 4]
```diff
@@ -57,7 +57,7 @@
     <div class="col-md-4 col-xs-12">
     </div>
     <div class="col-md-4 col-xs-12 chart" *ngIf="showGraph === graphType.Age && analyses.ageAnalysis" >
-      <app-chart [analysis]="analyses.ageAnalysis" [pointWidth]="20" [backgroundColor]="backgroundColor" [component]="component"> </app-chart>
+      <app-chart [analysis]="analyses.ageAnalysis" [pointWidth]="20" [backgroundColor]="backgroundColor" [domainType]="component"> </app-chart>
```
Comment (calbach) [inline public-ui/src/app/data-browser/concept-charts/concept-charts.component.html]: Would inline this one as well (or rename if needed)

**Comment by SrushtiGangireddy** [comment]
Thanks. Merging in.

**Pull request merged**